tasks/simulateScript.js

```diff
@@ -0,0 +1,31 @@
+const { simulateScript, decodeResult } = require("@chainlink/functions-toolkit")
+const path = require("path")
+const process = require("process")
+
+task("functions-simulate-script", "Initiates an on-demand request from a Functions consumer contract")
```

Comment:
task description needs to be "Locally simulates the execution of your custom source code"